docs/concepts/work-pools.md

```diff
@@ -570,3 +570,5 @@ The Prefect CLI can install the required package for Prefect-maintained worker t
 ### Additional resources
 
 See how to daemonize a Prefect worker in [this guide](/guides/deployment/daemonize/).
+
+For more information on overriding a work pool's job variables see this [guide](/overriding-job-variables).
```

Comment:
maybe we should prefix both of these with `/guides/` to be explicit

Reply:
done